ako-gateway-api/nodes/dequeue_ingestion.go

```diff
@@ -215,16 +215,22 @@ func saveAviModel(modelName string, aviGraph *nodes.AviObjectGraph, key string)
 func (o *AviObjectGraph) ProcessRouteDeletion(key, parentNsName string, routeModel RouteModel, fullsync bool) {
 
 	parentNode := o.GetAviEvhVS()
+	routeTypeNsName := routeModel.GetType() + "/" + routeModel.GetNamespace() + "/" + routeModel.GetName()
 
-	found, childVSNames := akogatewayapiobjects.GatewayApiLister().GetRouteToChildVS(routeModel.GetType() + "/" + routeModel.GetNamespace() + "/" + routeModel.GetName())
+	found, childVSNamesTemp := akogatewayapiobjects.GatewayApiLister().GetRouteToChildVS(routeTypeNsName)
+	childVSNames := make([]string, len(childVSNamesTemp))
+
+	copy(childVSNames, childVSNamesTemp)
```

Comment:
i don't see another reference to childVSNamesTemp

Reply:
as discussed, this task can be taken separately

Reply:
As per the discussion, have created https://avinetworks.atlassian.net/browse/AV-217196 ticket to make changes in all the functions of type GetXToY